governance/meetings/2024.md

```diff
@@ -0,0 +1,1491 @@
+* Check ID PR  
+  * [https://github.com/ossf/scorecard/pull/4021](https://github.com/ossf/scorecard/pull/4021)  
+  * Spener to make an issue about current breaking cron output changes  
+  * Allen can back out the cron specific output so we can merge partially and coordinate the rest in the issue
+
+## Jul 25, 2024 (Pacific)
+
+Attendees
+
+* Lelia Bray-Musso (Cisco)  
+* Dana Wang (OpenSSF)  
+* Spencer Schrock (Google)  
+* Caroline Lee (IBM)  
+* Anjlica Malla (Goldman Sachs)  
+* Jeff Mendoza (Kusari)  
+* Sarah Elder (NCSU)  
+* Stephen Augustus (Cisco)
+
+New Attendees
+
+* Nell Shamrell-Harrington (Microsoft)
+
+Regrets
+
+* Ulises Gascón (One Beyond)  
+* Teba Gomez (Whalar)
+
+Announcements
+
+* \[Spencer\] Scorecard [V5 launched](https://github.com/ossf/scorecard/releases/tag/v5.0.0). Scorecard Action release today/tomorrow   
+* \[Jeff\] Allstar is updating to scorecard.Run() before releasing with V5  
+  * \[Stephen\] Also trying to improve operations for both Scorecard and Allstar  
+  * Feedback for how these products are/aren’t working for you: [https://github.com/ossf/allstar/issues/528](https://github.com/ossf/allstar/issues/528)
+
+Agenda
+
+* \[Nell\] Proposal to add checks for Scorecard for different language ecosystems with re: to memory safety best practices  
+  * Some prior discussion on this [GitHub issue](https://github.com/ossf/scorecard/issues/3736), but the conversation went stale some time ago  
+  * Want to know whether this is something we would like the Memory Safety SIG to continue to pursue or something the SIG should bring to Scorecard later  
+  * Have been compiling a list of best practices for memory safe / non memory safe by default languages.  
+  * Active work on increasing configurability of Scorecard, getting away from requiring every repo to run every check, moving towards probes  
+  * Doing analysis on memory safety makes sense, probes may be a good way to play around with that, Spencer can point in the right direction on how to get started  
+    * Independent Probes  
+      * [https://github.com/ossf/scorecard/pull/4052](https://github.com/ossf/scorecard/pull/4052)   
+  * Non-goal to suggest rewriting a project in a memory-safe language  
+  * To make this a maintainable reality, would require language / ecosystem SMEs in order to achieve parity  
+  * Additional resources:  
+    * [https://github.com/ossf/Memory-Safety/blob/main/docs/best-practice-memory-safe-by-default-languages.md](https://github.com/ossf/Memory-Safety/blob/main/docs/best-practice-memory-safe-by-default-languages.md)   
+    * [https://github.com/ossf/Memory-Safety/blob/main/docs/best-practice-non-memory-safe-by-default-languages.md](https://github.com/ossf/Memory-Safety/blob/main/docs/best-practice-non-memory-safe-by-default-languages.md)   
+* \[Lelia\] Do we need to fix the instructions [at the top of this doc](#meeting-info) re: google group to join?  
```

Comment:
"at the top of this doc"

This link also doesn't work, so probably can unlink here too?